lib/kubernetes-deploy/kubernetes_resource/daemon_set.rb

```diff
@@ -38,6 +42,31 @@ def print_debug_logs?
 
     private
 
+    class Node
+      attr_reader :name
+
+      class << self
+        def kind
+          name.demodulize
+        end
+      end
+
+      def initialize(definition:)
+        @name = definition.dig("metadata", "name").to_s
+        @definition = definition
+      end
+    end
+
+    def find_nodes(cache)
+      # FIXME: should this also exclude tainted nodes?
```

Comment:
NIT - remove this before merging, i don't think we care about tainted nodes

Reply:
Agreed, we don't care about tainted nodes. If the pod was scheduled, we should assume it was supposed to be.